scraper/doctolib/doctolib_center_scrap.py

```diff
@@ -30,35 +31,40 @@
 
 DOCTOLIB_DOMAINS = DOCTOLIB_CONF.get("recognized_urls", [])
 
-
 DOCTOLIB_WEIRD_DEPARTEMENTS = SCRAPER_CONF.get("dep_conversion", {})
 
-
 logger = get_logger()
 booking_requests = {}
 
 
+def run_departement_scrap(departement: str):
+    logger.info(f"[Doctolib centers] Parsing pages of departement {departement} through department SEO link")
+    centers_departements = parse_pages_departement(departement)
+    if centers_departements == 0:
+        raise Exception("No Value found for department {}, crashing")
+    return centers_departements
+
+
 def parse_doctolib_centers(page_limit=None) -> List[dict]:
     centers = []
     unique_center_urls = []
 
-    for departement in get_departements():
-        logger.info(f"[Doctolib centers] Parsing pages of departement {departement} through department SEO link")
-        centers_departements = parse_pages_departement(departement)
-        if centers_departements == 0:
-            raise Exception("No Value found for department {}, crashing")
-        centers += centers_departements
+    with multiprocessing.Pool(50) as pool:
+        center_lists = pool.imap_unordered(run_departement_scrap, get_departements())
+        centers = []
 
-    centers = list(filter(is_vaccination_center, centers))  # Filter vaccination centers
-    centers = list(map(center_reducer, centers))  # Remove fields irrelevant to the front
+        for center_list in center_lists:
+            centers.extend(center_list)
```

Comment:
je pense que tu peux faire centers = list(center_list) pour avoir une liste directement à partir de l'itérateur

Reply:
Y'a aussi ça pour l'ajout d'un centre : liste_centres.append({**info_center, **dict_infos_browse_page}) . Pourquoi remettre les infos dans cet ordre ?

Reply:
En quoi l’ordre de la liste des centres a une importance ?